cg/apps/demultiplex/sample_sheet/api.py

```diff
@@ -187,6 +187,8 @@ def get_or_create_sample_sheet(self, flow_cell_name: str, bcl_converter: str | N
     def get_or_create_all_sample_sheets(self):
         """Ensure that a valid sample sheet is present in all flow cell directories."""
         for flow_cell_dir in self.flow_cell_runs_dir.iterdir():
+            if not flow_cell_dir.is_dir():
+                continue
```

Comment:
there is a utils function that gets directories specifically. Maybe use this here instead?